Commands/Public/room.js

```diff
@@ -0,0 +1,195 @@
+		} else {
+			await question.edit({
+				embed: {
+					color: Colors.PROMPT,
+					title: "I'll keep the room intact 😅",
+					description: "Do you want to add any members to this channel?",
+					footer: {
+						text: "You have 1 minute to respond.",
+					},
+				},
+			});
+			response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+			if (response) {
+				try {
+					await response.delete();
+				} catch (err) {
+					// No-op
+				}
+			}
+			if (response && configJS.yesStrings.includes(response.content.toLowerCase().trim())) {
+				await question.edit({
+					embed: {
+						color: Colors.PROMPT,
+						description: "Please send the names of the members you'd like to add.",
+						footer: {
+							text: "You can separate them with '|'",
+						},
+					},
+				});
+				response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+				try {
+					await response.delete();
+				} catch (err) {
+					// No-op
+				}
+
+				const failed = [];
+				const members = ArgParser.parseQuoteArgs(response.content, response.content.includes("|") ? "|" : " ");
+				await Promise.all(members.map(async memberQuery => {
+					let member;
+					try {
+						member = await client.memberSearch(memberQuery, msg.guild);
+					} catch (err) {
+						return failed.push(memberQuery);
+					}
+					if (!member || !member.user) return;
+					msg.channel.updateOverwrite(member, {
+						VIEW_CHANNEL: true,
+					}, `Room Management | Command issued by ${msg.member.tag}`).catch(err => {
+						winston.debug(`Failed to add member '${member.user.username}' to room '${msg.channel.name}' on server '${msg.guild.name}'`, { svrid: msg.guild.id, chid: msg.channel.id, usrid: member.id, err });
+					});
+				}));
+				question.edit({
+					embed: {
+						color: Colors.SUCCESS,
+						description: "Welcome, new people! 😘",
+						footer: !failed.length ? undefined : {
+							text: `I couldn't find members for ${failed.splice(0, 3).join(", ")}${failed.length > 3 ? " and more" : ""}`,
+						},
+					},
+				});
+			} else {
+				question.edit({
+					embed: {
+						color: Colors.SUCCESS,
+						description: "Ok! I won't add any members either.",
+					}
+				});
+			}
+		}
+	} else if (msg.suffix) {
+		const args = ArgParser.parseQuoteArgs(msg.suffix);
+		if (["text", "voice"].includes(args[0].toLowerCase())) {
+			const type = args[0];
+			args.splice(0, 1);
+			const members = [];
+
+			await Promise.all(args.map(async memberQuery => {
+				const member = await client.memberSearch(memberQuery, msg.guild).catch(() => {
+					// No-op
+				});
+				if (member) members.push(member);
+			}));
+
+			if (!serverDocument.config.room_category || !msg.guild.channels.has(serverDocument.config.room_category)) {
+				const categoryChannel = await msg.guild.channels.create("Talk Rooms", {
```

Comment:
I am not... keen on hard coding the channel name really. Then again, it doesn't matter. Talk Rooms is pretty not-good tho. Maybe just `Rooms`

Reply:
The feature is called "Talk Rooms". Additionally, the category ID is stored on the document so the category can be customized fully by admins.